### ISSUE TITLE
Feature request: add the ability to grab maximized windows from anywhere on the top panel, like in GNOME 

### ISSUE THREAD

**Issue by anticlam** [pop-os/cosmic-epoch#478]
This is very convenient (especially with browsers) and makes it so that I don't have to aim when I want to resize windows. It's the reason I can't switch away from gnome. To clarify, I'm referring to sliding on the universal top panel, not the one for just the window itself. Thanks.

https://github.com/user-attachments/assets/fa6a0c5c-5022-4592-b7f9-21424f220af1

**Comment by jacobgkau** [comment]
This would probably require work in cosmic-comp and/or cosmic-panel.

**Comment by feyerbrand** [comment]
Adding the ability to use corners for window management would be fantastic like The [WinTile Extenion](https://extensions.gnome.org/extension/1723/wintile-windows-10-window-tiling-for-gnome/) for Gnome

![image](https://github.com/user-attachments/assets/097b7971-b5aa-42ba-8200-38908a095f59)

**Comment by jacobgkau** [comment]
@feyerbrand Hotkeys for moving windows into corners is a separate topic from mouse-dragging functionality for un-maximizing windows. Please create a separate issue.

**Comment by amix3** [comment]
https://github.com/pop-os/cosmic-panel/issues/221

same request.

**Comment by Drakulix** [comment]
> @feyerbrand Hotkeys for moving windows into corners is a separate topic from mouse-dragging functionality for un-maximizing windows. Please create a separate issue to discuss that one. (I hid your comment simply because the GIF is large and easily distracts from the earlier video.)
> 
> We do already have mouse snapping into corners.

We already even have shortcuts in floating mode to do this. Super+Shift+Arrows (or +HJKL) by default.

**Comment by Drakulix** [comment]
> This would probably require work in cosmic-comp and/or cosmic-panel.

Yes, detecting the grab in the panel is easy, but we probably would need a new protocol or similar to initiate a grab on another window from the panel.
